src/video_core/buffer_cache/buffer_cache.cpp

```diff
@@ -288,6 +302,9 @@ u32 BufferCache::BindIndexBuffer(bool& is_indexed, u32 index_offset) {
         cmdbuf.bindIndexBuffer(stream_buffer.Handle(), offset, index_type);
         return new_index_size / index_size;
     }
+    if (regs.primitive_type == AmdGpu::PrimitiveType::Polygon) {
+        UNREACHABLE();

```

Comment:
Not sure if you want to tackle this now but I think it should be trivial to implement, basically translating an indexed draw is the same except instead of outputting `i` for example you would output `input_indices[i]`. See how `ConvertQuadToTriangleListIndices` is very similar to `EmitQuadToTriangleListIndices`.

Reply:
I'd rather wait until I can reach this unreachable as not many games use this primitive